grpc/orderbook.go

```diff
@@ -23,6 +24,9 @@ func NewOrderbookClient() orderbook.Client {
 
 // OpenOrder implements the orderbook.Client interface.
 func (client *orderbookClient) OpenOrder(ctx context.Context, multiAddr identity.MultiAddress, orderFragment order.EncryptedFragment) error {
+	if orderFragment.IsEmpty() {
```

Comment:
Again, in general consider the naming of these methods to be more representative of the internal check — `IsNil`.